setup.cfg

```diff
@@ -31,20 +31,16 @@ exclude =
   .Python
 
 # Things to ignore:
-#   E251: spaces around default arg value (against our style)
 #   E128: continuation line under-indented (too noisy)
-#   F401: unused identifiers (useless, as it doesn't see inside # type: comments)
 #   W601: has_key() deprecated (false positives)
 #   E701: multiple statements on one line (colon) (we use this for classes with empty body)
-#   W503: line break before binary operator
-#   W504: line break after binary operator
```

Comment:
by switching from `ignore` to `extend-ignore` (new in flake8 3.7) you no longer have to re-enable the default-ignored things